docs/dunst.5.pod

```diff
@@ -64,47 +64,59 @@ keyboard focus.
 
 =back
 
-=item B<geometry> (format: [{width}][x{height}][+/-{x}[+/-{y}]], default: "0x0+0-0")
+=item B<geometry> DEPRECATED
 
-The geometry of the window the notifications will be displayed in.
-
-=over 4
+This setting is deprecated. It's split up into B<width>, B<height>, B<origin>,
+B<notification_limit> and B<offset>.
 
 =item B<width>
 
-The width of the notification window in pixels. A negative value sets the width
-to the screen width B<minus the absolute value of the width>. If the width is
-omitted then the window expands to cover the whole screen. If it's 0 the window
-expands to the width of the longest message being displayed.
+The width of the notification window in pixels. This can be a single number to
+specify a constant width or two numbers for the minimum and maximum width. The
+notification will expand from the minimum width as neccesary.
 
-=item B<height>
+Examples:
+        width = 300      # constant width of 300
+        width = (0, 300) # width between 0 and 300
 
-The number of notifications that can appear at one time. When this
-limit is reached any additional notifications will be queued and displayed when
-the currently displayed ones either time out or are manually dismissed. If
-B<indicate_hidden> is true, then the specified limit is reduced by 1 and the
-last notification is a message informing how many hidden notifications are
-waiting to be displayed. See the B<indicate_hidden> entry for more information.
+=item B<height>
 
-The physical(pixel) height of the notifications vary depending on the number of
-lines that need to be displayed.
+The maximum height of a single notification.
 
-See B<notification_height> for changing the physical height.
+=item B<notification_limit> (default: 0)
 
-=item B<x/y>
+The number of notifications that can appear at one time. When this
+limit is reached any additional notifications will be queued and displayed when
+the currently displayed ones either time out or are manually dismissed. The
+value 0 means no limit. If B<indicate_hidden> is true, then the specified limit
+is reduced by 1 and the last notification is a message informing how many hidden
+notifications are waiting to be displayed. See the B<indicate_hidden> entry for
+more information.
+
+=item B<origin> (default: top-right)
+
+The origin of the notification window on the screen. It can then be moved with
+offset.
+Origin can be one of:
+        top-left
+        top-center
+        top-right
+        bottom-left
+        bottom-center
+        bottom-right
+        left-center
+        center
+        right-center
+
+=item B<offset> format: (horizontal, vertical)
```

Comment:
Looks like the format is wrong, should be like `10x300`

Reply:
Thank you! I've updated it. I think that immediately solves your other problem as well.